src/components/ContentModal.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="modal">
+    <div class="modal-content" @click.stop>
+      <p id="modalText" v-if="!isImage">
+        {{ cardContent.content }}
+      </p>
+      <img
+        alt="image"
+        id="modalImage"
+        :src="cardContent.content"
+        class="img-responsive"
+        v-if="isImage"
+      />
```

Comment:
Be uniform with using Kebab-case or CamelCase as id or class names.

Reply:
I will change them to CamelCase.